### PR TITLE
feat: Add date description to each release

### PR DESCRIPTION
**Pull request by vapurrmaid**
Resolves: #33

### CODE REVIEW

**Review by kylecarbs — COMMENTED** [review]
Maybe we add "Released on <date>" as a prefix? The dots also conflict with the title to me... Maybe we do `12/18/2020` or somethin...

Full statement: "Released on 12/18/2020"

**Review by khorne3 — APPROVED** [review]

**Comment by vapurrmaid** [comment]
> Maybe we add "Released on " as a prefix? The dots also conflict with the title to me... Maybe we do `12/18/2020` or somethin...
> 
> Full statement: "Released on 12/18/2020"

Yea totally down for that. I had adapted the dots from the blog style: https://coder.com/blog, but I myself prefer the prefix and slashes as well

File: changelog/1.13.2.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "1.13.2"
+description: "11.23.2020"
```
Comment (vapurrmaid) [inline changelog/1.13.2.md:3]:
```suggestion
description: "Released on 11/23/2020"
```

File: changelog/1.14.0.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "1.14.0"
+description: "12.18.2020"
```
Comment (vapurrmaid) [inline changelog/1.14.0.md:3]:
```suggestion
description: "Released on 12/18/2020"
```

File: changelog/1.14.1.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "1.14.1"
+description: "12.18.2020"
```
Comment (vapurrmaid) [inline changelog/1.14.1.md:3]:
```suggestion
description: "Released on 12/18/2020"
```

File: changelog/1.14.2.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "1.14.2"
+description: "12.22.2020"
```
Comment (vapurrmaid) [inline changelog/1.14.2.md:3]:
```suggestion
description: "Released on 12/22/2020"
```

**Pull request merged**